gamest_plugins/process_identifier/module.py

```
@@ -113,25 +114,37 @@ def get_settings_template(cls):
 
         return d
 
+    def _iter_survivors(self):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_iter_survivors`

(ANN202)